packages/dev/docs/src/utils.js

```diff
@@ -39,3 +39,15 @@ export function getUsedLinks(obj, links, usedLinks = {}) {
 
   return usedLinks;
 }
+
+export function getAnchorProps(href) {
```

Comment:
in theory we can use this function everywhere -- happy to do so if people think i should?